sysutils/fx/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        antonmedv fx 18.0.1
+
+categories          sysutils
+platforms           darwin
+license             MIT
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         Command-line tool and terminal JSON viewer
+long_description    ${description}
+
+checksums           rmd160  86d17a60e47f88a334292566ae584af06c08343e \
+                    sha256  a0e6bf4bac205a5b04b9553d47692a6b80b9fc34e2b679c8472b70686ca78cd5 \
+                    size    11106322
+
+master_sites        ${github.homepage}/releases/download/${version}
+distname            ${name}-macos
+
+extract.mkdir       yes
+
+use_configure       no
+use_zip             yes
+
+build {}
```

Comment:
No, that's archive with precompiled binary file.

Reply:
Can it be built from source? We prefer to do that if possible.

Reply:
That's not so easy, at least I don't know how to build it properly within macports. since it's JavaScript project, so we need to prefetch all npm dependencies in order to build it, keep their checksums and so on. I would prefer to use official prebuilt binary.

Reply:
Then you need to indicate the architectures for which that binary was built in the `supported_archs` line. If the software uses C++, you'll need to indicate which C++ standard library it was built for using the `configure.cxx_stdlibrary` line. And assuming that the binary is not compatible with all older releases of macOS, you'll need to add code to prevent installation on OS versions on which it won't work.